app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -279,7 +309,16 @@ const MakeDecision = ({ data }) => {
             {formatMessage(messages.machineLearningSuggestions)}
           </div>
         ),
-        content: null,
+        content: (
+          <MachineLearningSuggestions
+            items={fakeData}
+            modalState={modalState}
+            setModalState={setModalState}
+            itemData={itemData}
+            collapseTabsExceptCurr={collapseTabsExceptCurr}
+            isNarrowView={!collapsedRightSection}
+          />
+        ),
```

Comment:
Please check this section on step and log view.

Reply:
I take mock data from historyItems, which are not rendered from step level, only from log. I waiting for the backend data. Then i can display data in step and log view